apparelo/apparelo/doctype/item_production_detail/item_production_detail.py

```diff
@@ -224,8 +224,9 @@ def create_process_details(self):
 								cutting_attribute=dict(counter_attr)
 								for value in cutting_attribute:
 									cutting_attribute[value]=list(set(cutting_attribute[value]))
-					process_variants['variants'] = list(set(variants_))
-					process_variants['BOM']=list(set(boms))
+					
+					process_variants['variants'] = list(set([var for var in variants_ if not ',' in var] ))
```

Comment:
same here

Reply:
## 800234b613a40f904c08949dcb001c68a8cc7f54
The above commit resolve all the changes requested